test/unit/sessions/softphone-session-handler.test.ts

```diff
@@ -461,7 +461,7 @@ describe('proceedWithSession()', () => {
     getUserParticipantFromConversationEventSpy.mockReturnValue(participant);
 
     handler.activeSession = new MockSession() as any;
-    handler.activeSession.id = pendingSession.id;
+    handler.activeSession!.id = pendingSession.id;
```

Comment:
Codacy has a fix for the issue: [Forbidden non-null assertion.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9637746)
```suggestion
    handler.activeSession?.id = pendingSession.id;
```